pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTest.java

```diff
@@ -384,8 +385,11 @@ public void testReplication(String namespace) throws Exception {
         consumer3.receive(1);
     }
 
-    @Test
+    @Test(invocationCount = 5)
     public void testReplicationWithSchema() throws Exception {
+        config1.setBrokerDeduplicationEnabled(true);
+        config2.setBrokerDeduplicationEnabled(true);
+        config3.setBrokerDeduplicationEnabled(true);
```

Comment:
I thought the deduplication is enabled for geo-replication by default, but not actually.
Without this configuration, the test might get duplicated messages because we will rewind the cursor
after the schema is fetched.

And it also should be problem for geo-replication that the producer also can disconnect to the remote
cluster and introduce the duplicated message.

IMO, we should enable the data deduplication by default for geo-replication. It should be discussed under the
mailing list first, so I don't want to have this change in this PR.

Reply:
I agree